.env.example

```
@@ -1,14 +1,49 @@
+#   sudo systemctl start mysqld
+#   sudo mysql -e "CREATE DATABASE earn_db"
 
-# option 2: PlanetScale Database
-# use this if you're connecting to PlanetScale
-# format: mysql://USERNAME:PASSWORD@HOST/DATABASE_NAME?sslaccept=strict
-#DATABASE_URL=''
+# MySQL instance in Docker (Only if docker is installed)
+#   docker pull mysql:latest
+#   docker run --name mysql-container -e MYSQL_ROOT_PASSWORD=my-secret-pw -d -p 3306:3306 mysql:latest
+#   Check if a local container has started in your docker desktop
+DATABASE_URL="mysql://root:my-secret-pw@localhost:3306/mydb"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)